create-mern-ts-app.js

```diff
@@ -43,6 +44,9 @@ function createProjectTemplate(projectName) {
   console.log(chalk.cyan(destRoot + '\n'));
   fs.mkdirsSync(destRoot);
   fs.copySync(srcRoot, destRoot);
+  let envContents = fs.readFileSync(path.join(destRoot, 'backend', '.env.example'), 'utf8')
+    .replace(/auth-shared-secret/g, crypto.randomBytes(48).toString('hex'));
```

Comment:
I think some `24` bytes would already be quite enough here!

Reply:
Done!